general-concepts.md

```diff
@@ -1,53 +1,51 @@
-?> _TODO_  This page is in *Brazilian Portuguese* only so far. **If you can, please help us translate it to *English*.**
+# General Concepts
 
-# Conceitos Gerais
-
-### Vídeo Tutorial: Usando o Tainacan - Conceitos gerais
+### Video Tutorial: Using Tainacan - General Concepts (Portugeuse)
 <iframe
     width="560"
     height="513" 
-    src="https://www.youtube.com/embed/fzbd36fbuII?start=52"
+    src="https://www.youtube.com/watch?v=fzbd36fbuII"
```

Comment:
I would suggest keeping the same link here, unless you have one particular reason for it, ok?